sql/sys_vars.cc

```diff
@@ -8708,6 +8708,20 @@ static Sys_var_bool Sys_pxc_encrypt_cluster_traffic(
     READ_ONLY GLOBAL_VAR(pxc_encrypt_cluster_traffic), CMD_LINE(OPT_ARG),
     DEFAULT(true), NO_MUTEX_GUARD, NOT_IN_BINLOG);
 
+static const char *wsrep_encrypt_modes[] = {"OFF", "ON", "NONE",
+                                        NullS};
+static Sys_var_enum Sys_wsrep_gcache_encrypt(
+    "wsrep_gcache_encrypt", "Encrypt GCache. This variable is solely for testing purposes. "
```

Comment:
Do we plan to keep this variable when we release it? Shouldn't it be behind some testing/internal ifdef?